# -*- coding: utf-8 -*-
# Generated by Django 1.10.1 on 2017-06-19 12:55
from __future__ import unicode_literals

from django.db import migrations


class Migration(migrations.Migration):

    dependencies = [
        ('api', '0002_auto_20170619_1243'),
    ]

    # gives error: django.db.utils.OperationalError: (1071, 'Specified key was too long; max key length is 767 bytes')
    # need to possibly resolve using https://stackoverflow.com/questions/29782081/django-db-utils-operationalerror-1071-specified-key-was-too-long-max-key-le
    # however, no direct access to DB using dokku
    # operations = [
    #     migrations.AlterUniqueTogether(
    #         name='identity',
    #         unique_together=set([('identifier', 'environment')]),
    #     ),
    # ]
